README.md

```diff
@@ -88,6 +88,25 @@ library implemented in Java, with ports to other languages.
 Post to the [discussion forum](https://groups.google.com/group/zxing) or tag a question with [`zxing`
 on StackOverflow](https://stackoverflow.com/questions/tagged/zxing).
 
+## License
```

Comment:
This is already in the standard LICENSE file in the repo and also shown by Github; I'd rather not add more to this file to duplicate it.

Reply:
The LICENSE file is a literal text of the Apache license with an appendix (in my opinion irrelevant and unnecessary) about imagio library (which is not part of the source code and hence doesn’t need to be mentioned).

However, the usual convention is that a separate copyright and license statement is a good thing to make it clear who owns the copyrights and what the licenses are. Also, not everyone receives the library from GitHub and is able to view the header on the page for various reasons.

Reply:
I'm following ASF conventions here for including license info: https://www.apache.org/legal/resolved.html The license from third party libraries does need to be reproduced somewhere in distributions, and there are a bunch of ways to do it, but this seems like a fine standard to follow. 

It's already labeled in metadata, pom.xml, that is read by Github and other tools, for anything that wants something machine-readable.

What problem are you solving here ...